openapi.yaml

```diff
@@ -1660,6 +1660,17 @@ paths:
                   - processes
                   - links
                 properties:
+                  version:
+                    type: string
+                    nullable: true
+                    description: >-
+                      Version number of the openEO process specification this back-end
+                      implements.
+                      If not provided or `null` the version is either unknown or multiple versions are implemented.
+                      Clients must be able to handle multiple process versions at the same time.
```

Comment:
I guess you mean here multiple versions of the openeo-processes specification, not multiple process versions at the same time (a backend can only define a single specification per process)

Reply:
```suggestion
                      Clients must be able to handle multiple process specification versions at the same time.
```